### DISCUSSION TITLE
Atari Adapters?

### DISCUSSION THREAD

**Discussion by 7ank0v1c** [sanni/cartreader#517] [Need help]
Hey guys, I was wondering if there is any Atari adapters for the Sanni? I have a MS.Pacman for the 2600 I would like to dump, and I also want to purchase the original Mario Bros. for the 2600 to dump also?

I love this project btw, I just received my pre-build sanni yesterday, and love it already. It is far superior to the Retrode 2 and Kazoo Nes Dumper...

**Comment by PW5190** [comment]
I'm waiting for cartridge adapters for every cartridge-based Atari system, aside from 2600. Hopefully we can get adapters for 5200, Jaguar and Lynx soon, as well as a 7800 adapter. After all, the 7800 is backward compatible with the 2600, and so the 2600 and 7800 adapter could be together in one piece.
